helpers/test-helper.js

```diff
@@ -1272,13 +1274,25 @@ exports.getMultichainSkillId = function getMultichainSkillId(chainId, skillId) {
   return ethers.BigNumber.from(chainId).mul(ethers.BigNumber.from(2).pow(128)).add(ethers.BigNumber.from(skillId));
 };
 
-exports.upgradeColonyTo = async function (colony, _version) {
-  const version = new BN(_version);
-  let currentVersion = await colony.version();
-  while (currentVersion.ltn(version)) {
-    await colony.upgrade(currentVersion.addn(1));
-    currentVersion = await colony.version();
-  }
+exports.upgradeColonyOnceThenToLatest = async function (colony) {
```

Comment:
I would elaborate more here. It seems like the idea is that we want to test upgrading from a specific version, since that version had some bug or something we want to make sure we recover from. But aside from the specific version we want to test, we don't care about other versions, so we can just "fast forward" through them.